src/ui/LanguagesModel.cpp

```diff
@@ -0,0 +1,77 @@
+/* Ricochet - https://ricochet.im/
+ * Copyright (C) 2014, John Brooks <john.brooks@dereferenced.net>
```

Comment:
If you like, you can put your own name here. You retain copyright on the code you submit.